presto-docs/src/main/sphinx/release/release-332.rst

```diff
@@ -59,10 +59,10 @@ Hive Connector Changes
   from partitioned tables would return zero rows, and reading from
   unpartitioned tables would fail with a cryptic error. (:issue:`3366`)
 * Add ``hive.fs.new-directory-permissions`` configuration property for setting the permissions of new directories
-  created by Presto. Default value is ``0777``. (:issue:`3126`)
+  created by Presto. Default value is ``0777``, which corresponds to previous behaviorrecate the. (:issue:`3126`)
```

Comment:
The last part of the sentence is mangled

Reply:
fixed